scripts/tasks/jest.ts

```diff
@@ -22,7 +22,7 @@ const commonArgs = (): JestTaskOptions => {
     // Just specific config
     nodeArgs: args.nodeArgs,
     // pass forward positional args (to narrow down tests to be run)
-    _: args._,
+    // _: args._, // TODO: investigate and fix. this broke `update-snapshots`.
```

Comment:
This should be the real fix. There was similar logic before but it was removed in the recent jest task update.
```suggestion
    _: args._.filter(arg => arg !== 'jest' && arg !== 'jest-watch'),
```

Reply:
**thought:** 
passing through random positional args leads to unexpected bugs and I suggest to remove it if we'll not change dev tooling anytime soon. (all of this happening as there is not 1 correct way to invoked underlying abstraction...)

To make this a proper "fix" one should also filter out for example `test` and maybe various other just defined task names that exist/might be added in the future. 
> Why?
>``` 
> npm script: -->  "test": "just-scripts test" 
>       --> `yarn  test -u` will pass down `test` as positional 
>            --> ERROR 
> ```

Overall just another reason why I don't think this is good enough tooling going forward :)